workers/social/lib/social/main.coffee

```diff
@@ -138,7 +138,7 @@ do ->
 
   options = { rateLimitOptions : KONFIG.nodejsRateLimiter }
 
-  app.post '/remote.api/:model/:id?', (require './remoteapi') koding
+  app.post '/remote.api/:token?/:model/:id?', (require './remoteapi') koding
```

Comment:
yeah there is no other way around :| but we do support both in Header and in-Route at least. maybe we can pinpoint locations where we are logging and shadow them but I don't think it would be a big problem.